website/content/docs/k8s/k8s-cli.mdx

```diff
@@ -2,73 +2,76 @@
 layout: docs
 page_title: Consul K8s CLI Reference
 description: >-
-  Consul K8s CLI is a tool for quickly installing and interacting with Consul on Kubernetes.
+  The Consul on Kubernetes CLI is a tool for installing and managing Consul on Kubernetes.
 ---
 
-# Consul K8s CLI Reference
+# Consul on Kubernetes CLI Reference
 
-Consul K8s CLI is a tool for quickly installing and interacting with Consul on Kubernetes. 
-The Consul K8s CLI allows you to manage the lifecycle of Consul without requiring the usage of `Helm`, [Consul CLI](/commands/index), and `kubectl`. 
-The Consul K8s CLI offers a Kubernetes native experience for managing Consul. 
+The Consul on Kubernetes CLI, `consul-k8s`, is a tool for managing Consul
+without needing `Helm`, the [Consul CLI](/commands/index), or `kubectl`.
 
--> **Note**: For guidance on how to install the Consul K8s CLI, visit the [Installing the Consul K8s CLI](/docs/k8s/installation/install-cli) documentation. 
+-> **Note**: For guidance on how to install `consul-k8s`, visit the
+[Installing the Consul K8s CLI](/docs/k8s/installation/install-cli) documentation.
 
-This topic describes the subcommands and available options for using Consul K8s CLI.
+This topic describes the commands and available options for using `consul-k8s`.
 
 ## Usage
 
-Consul K8s CLI uses the following syntax:
+The Consul on Kubernetes CLI uses the following syntax:
 
 ```shell-session
-$ consul-k8s <SUBCOMMAND> <OPTIONS>
+$ consul-k8s <COMMAND> <OPTIONS>
 ```
 
-## Subcommands
+## Commands
 
-You can use the following subcommands with `consul-k8s`.
+You can use the following commands with `consul-k8s`.
 
-  - [install](#install)
-  - [uninstall](#uninstall)
-  - [status](#status)
-  - [upgrade](#upgrade) <sup>BETA</sup>
-  - [version](#version) 
+  - [`install`](#install): Install Consul on Kubernetes.
+  - [`proxy`](#proxy): Inspect Envoy proxies managed by Consul.
+      - [`proxy list`](#proxy-list): List all Pods running proxies managed by Consul.
+      - [`proxy read`](#proxy-read): Inspect the Envoy configuration for a given Pod.
+  - [`status`](#status): Check the status of a Consul installation on Kubernetes.
+  - [`uninstall`](#uninstall): Uninstall Consul deployment.
+  - [`upgrade`](#upgrade): Upgrade Consul on Kubernetes from an existing installation.
+  - [`version`](#version): Print the version of the Consul on Kubernetes CLI.
```

Comment:
I added descriptions to the TOC so that readers can see what the commands do at a glance.